pkg/scheduler/objects/queue.go

```diff
@@ -656,7 +656,10 @@ func (sq *Queue) addChildQueue(child *Queue) error {
 		}
 		// managed (configured) leaf queue can't use template
 	} else {
-		child.template = sq.template
+		// don't override the template of non-leaf queue
+		if child.template == nil {
```

Comment:
Please add logging for this case, something like inheriting child template for queue...

Reply:
done!